osmaxx/core/templates/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Osmaxx</title>
+    <link rel="icon" type="image/png" href='{% static "osmaxx/images/favicon.png" %}' />
```

Comment:
The general idea is to easily apply a different *base* by providing (read: overriding) the default `base.html` with minimal interference. 
Good idea to move that part down the cascade.